x-pack/plugins/cases/public/components/markdown_editor/eui_form.tsx

```diff
@@ -60,6 +71,39 @@ export const MarkdownEditorForm = React.memo(
         [id, field.value, caseTitle, caseTags]
       );
 
+      useEffect(() => {
+        const storageDraftComment = draftCommentStorageKey && storage.get(draftCommentStorageKey);
+        if (storageDraftComment && storageDraftComment !== '') {
+          field.setValue(storageDraftComment);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+      }, []);
+
+      useEffect(() => {
+        if (initialValue && initialValue !== field.value) {
+          field.setErrors([{ message: i18n.COMMENT_VERSION_CONFLICT_ERROR }]);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+      }, [initialValue]);
+
+      useDebounce(
+        () => {
+          if (isFirstRender.current) {
```

Comment:
Just curious, how come we want to ignore the first render? Is it because it would store the initial value before the user starts typing?

Could we instead just compare the `field.value` with `initialValue` and see if they're equal?

Reply:
yes, every time you visit a case, you get an empty newComment key stored in the session eventhough user didn't touch the markdown.
i.e. `xpack.cases.caseView.{caseId}.{newComment}.markdownEditor`
So we want to ignore first render so that it doesn't store `newComment` markdown key with empty value. 

Unfortunately `initialValue` is `undefined` for `newComment` markdown until `submit(add comment)` is triggered. That's why `initialValue` is not helpful to detect first render.